src/configuration/console-config.tsx

```diff
@@ -96,87 +95,81 @@ export const ConsoleConfigPage: FC<ConsoleConfigProps> = ({ brokerId }) => {
   }
 
   return (
-    <Stack key={'stack' + brokerId}>
-      <StackItem isFilled>
-        <Flex>
-          <FlexItem>
-            <FormGroup
-              label="Expose"
-              fieldId={'console-config-expose-formgroup'}
-              key={'formgroup-console-expose'}
-            >
-              <Checkbox
-                label="Expose Console"
-                isChecked={exposeConsole}
-                name={'check-console-expose'}
-                id={'check-expose-console'}
-                onChange={setConsoleExpose}
-              />
-            </FormGroup>
-          </FlexItem>
-          <FlexItem>
-            <FormGroup
-              label="ExposeMode"
-              fieldId={'console-config-exposemode-formgroup'}
-              key={'formgroup-console-exposemode'}
+    <Form isHorizontal isWidthLimited key={'form' + brokerId}>
+      <Grid hasGutter md={6}>
+        <FormFieldGroup>
+          <FormGroup
+            label="Expose"
```

Comment:
Hey Thomas...all changes looks good!
just a small suggestion here can we use useTranslation() hook to change the `label` text aswell?

Reply:
As discussed on slack we will use a tool to extract the translations for us. I'm resolving the conversation.